### PR TITLE
feat: support `innerText` locators

### PR DESCRIPTION
**Pull request by sadym-chromium**
Start implement innerText locator in [`browsingContext.locateNodes`](https://w3c.github.io/webdriver-bidi/#commands-browsingcontextlocatenodes) using WebAPI.

Out of scope:
* `maxNodeCount`
* `maxDepth`

### CODE REVIEW

**Comment by sadym-chromium** [comment]
Actually, the `innerText` selector cannot be implemented via xPath `text()`, as the latest does not respect the text of the nested nodes.

**Review by OrKoN — APPROVED** [review]

File: src/bidiMapper/domains/context/BrowsingContextImpl.ts [hunk 1]
```diff
@@ -1032,48 +1032,116 @@ export class BrowsingContextImpl {
...
-          return array;
-        });
+        return {
+          functionDeclaration: String((cssSelector: string) => {
+            const results = document.querySelectorAll(cssSelector);
+            const array = [];
+            for (const item of results) {
+              array.push(item);
+            }
+            return array;
```
Comment (OrKoN) [inline src/bidiMapper/domains/context/BrowsingContextImpl.ts:1048]: Array.from(document.querySelectorAll(cssSelector))?

Comment (sadym-chromium) [reply src/bidiMapper/domains/context/BrowsingContextImpl.ts:1048]: Out of scope of this PR? I will rework it anyways to adopt other options.

Comment (OrKoN) [reply src/bidiMapper/domains/context/BrowsingContextImpl.ts:1048]: sg

**Pull request merged**